src/test/java/htsjdk/samtools/seekablestream/SeekableBufferedStreamTest.java

```diff
@@ -69,6 +70,20 @@ public void testRandomRead() throws IOException {
         assertEquals(buffer1, buffer2);
     }
 
+    @Test
+    public void testReadExactlyOneByteAtEndOfFile() throws IOException {
+        try( SeekableStream stream = new SeekableHTTPStream(new URL(BAM_URL_STRING))){
```

Comment:
odd whitespace here, I would expect to see `try (SeekableStream stream = new SeekableHTTPStream(new URL(BAM_URL_STRING))) {`

Reply:
updated